src/test/java/com/cityelf/utils/ParserGasTest.java

```diff
@@ -0,0 +1,57 @@
+package com.cityelf.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.cityelf.domain.ForcastData;
+
+import org.jsoup.Jsoup;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ParserGasTest {
+
+  @Autowired
+  private ParserGas parserGas;
+
+  private String content;
+
+  @Before
+  public void setUp() throws Exception {
+    File file = new File(".\\src\\test\\java\\com\\cityelf\\utils\\gasWebResponseTest.txt");
```

Comment:
You have to move this file into the test\resources folder. This folder is exactly for such files.
Look here how to read it then:
https://www.mkyong.com/java/java-read-a-file-from-resources-folder/